postprocessing/comstockpostproc/comstock_to_eia_comparison.py

```diff
@@ -93,8 +96,24 @@ def __init__(self, comstock_list: List[ComStock], eia_list: List[EIA], upgrade_i
             self.name = ' vs '.join(sorted(dataset_names,key=len)[:2])
 
         # Combine into a single dataframe for convenience
-        self.monthly_data = pd.concat(monthly_dfs_to_concat, join='outer', ignore_index=True)
-        self.data = pd.concat(annual_dfs_to_concat, join='inner', ignore_index=True)
+        # self.monthly_data = pl.concat(monthly_dfs_to_concat, join='outer', ignore_index=True)
+        self.monthly_data = pl.concat(monthly_dfs_to_concat, how="vertical_relaxed")
+        assert isinstance(self.monthly_data, pl.LazyFrame)
+
+        # self.data = pl.concat(annual_dfs_to_concat, join='inner', ignore_index=True)
+        common_columns = set(annual_dfs_to_concat[0].columns)
```

Comment:
That's my approach to implement the `join='inner'` , find all the shared columns and select them out.